airbyte-integrations/connectors/source-greenhouse/acceptance-test-config.yml

```diff
@@ -17,6 +19,11 @@ tests:
   basic_read:
     - config_path: "secrets/config.json"
       configured_catalog_path: "integration_tests/configured_catalog.json"
+      expect_records:
+        path: "integration_tests/expected_records.txt"
```

Comment:
the acceptance tests were generated from master

```
python main.py read --config secrets/config.json --catalog integration_tests/configured_catalog.json | grep '"type": "RECORD"' | jq -c '.record | del(.data.attachments)' > integration_tests/expected_records.txt
```
then removed .applications.attachments from the candidates stream's records